org-roam-graph.el

```diff
@@ -97,6 +97,12 @@ Example:
   :type 'number
   :group 'org-roam)
 
+(defcustom org-roam-graph-wrap-title t
```

Comment:
We should make the variable something like `org-roam-graph-title-shortener`, and the values be `'truncate`, `'wrap`, nil, and later below do a pcase on this to apply the appropriate function.

Reply:
what should be the default?

Reply:
the original (truncate), to minimize surprise.